pyigm/fN/tests/test_fnmodel.py

```diff
@@ -0,0 +1,60 @@
+# Module to run tests on FNModel
+
+# TEST_UNICODE_LITERALS
+
+import numpy as np
+import os, pdb
+from astropy import units as u
+
+from pyigm.fN.fnmodel import FNModel
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'files')
+    return os.path.join(data_dir, filename)
+
+
+def test_init():
+    # Class
+    fN_P13 = FNModel('Hspline', zmnx=(2.,5.))
+    # Mtype
+    assert fN_P13.mtype == 'Hspline'
+    # Class
+    fN_I14 = FNModel('Gamma')
+    # Mtype
+    assert fN_I14.mtype == 'Gamma'
+    assert fN_I14.zmnx == (0., 10.)
+
+
+def test_default():
+    fN_default = FNModel.default_model()
+    #
+    assert fN_default.mtype == 'Hspline'
+    assert fN_default.zmnx == (0.5, 3)
+
+
```

Comment:
I plan to add some additional tests once I ingest
some f(N) constraints.  Will do that once the FNModel
syntax settles.